[W13-2] EventsPlus+

kaitingpeck opened the pull request:
@cjiefeng
@pratyayj
@ryantanwk
@yap666y

asiddharth commented on README.adoc:
```diff
@@ -15,24 +14,36 @@ ifndef::env-github[]
 
 == Acknowledgements
 
+* This project is built on the AddressBook-Level4 project created by SE-EDU initiative at https://github.com/se-edu/
```
Good job on acknowledging AB4

asiddharth commented on README.adoc:
```diff
@@ -15,24 +14,36 @@ ifndef::env-github[]
 * <<UserGuide#, User Guide>>
 * <<DeveloperGuide#, Developer Guide>>
-* <<LearningOutcomes#, Learning Outcomes>>
 * <<AboutUs#, About Us>>
```
Good job here

asiddharth commented on docs/ContactUs.adoc:
```diff
@@ -4,4 +4,4 @@
 * *Bug reports, Suggestions* : Post in our https://github.com/se-edu/addressbook-level4/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
 * *Contributing* : We welcome pull requests. Follow the process described https://github.com/oss-generic/process[here]
-* *Email us* : You can also reach us at `damith [at] comp.nus.edu.sg`
+* *Email us* : You can also reach us at `damith@comp.nus.edu.sg`
```
Please change the contact us page to reflect the appropriate contacts (or remove the old ones).

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -229,11 +233,245 @@ Format: `exit`
+
+Format: `notification <EVENT INDEX> enable/disable`
+
+Examples:
```
Nice set of features !
You may want to add ‘coming in v2.0’ for features not yet implemented.

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -229,11 +233,245 @@ Format: `exit`
-_{explain how the user can enable/disable data encryption}_
-// end::dataencryption[]
+* `notification 2 disable`
+  V 1.1: Access Windows OS notification
```
Are these 4 lines intended to be here?

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -258,3 +496,40 @@ e.g.`select 2`
 * *History* : `history`
 * *Undo* : `undo`
 * *Redo* : `redo`
+* *Add Contacts to Group* : `addToGroup group/GROUP_NAME [i/INDEX]` +
```
Good job adding the command summary here!

asiddharth commented on src/main/java/seedu/address/logic/parser/AddressBookParser.java:
```diff
@@ -49,39 +49,51 @@ public Command parseCommand(String userInput) throws ParseException {
         switch (commandWord) {
 
         case AddCommand.COMMAND_WORD:
+        case AddCommand.COMMAND_WORD_ALIAS: // test case for command alias
```
Would it be better if each case in switch case is handled separately? Although correct, seems a bit confusing to understand here.

asiddharth replied:
Same comment

asiddharth commented on src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:
```diff
@@ -47,19 +47,41 @@ public void parseCommand_add() throws Exception {
+    @Test
+    public void parseCommand_delete_alias() throws Exception {
+        DeleteCommand command = (DeleteCommand) parser.parseCommand(
+                DeleteCommand.COMMAND_WORD_ALIAS + " " + INDEX_FIRST_PERSON.getOneBased());
```
Would it be better to assign a variable to 
DeleteCommand.COMMAND_WORD_ALIAS + " " + INDEX_FIRST_PERSON.getOneBased() 
for better readablility?

kaitingpeck replied:
will take note of this, thank you!

asiddharth commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,59 @@
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Jack Chai Jie Feng
+image::jack.png[width="150", align="left"]
+{empty}[https://github.com/cjiefeng[github]]
```
May be helpful to add the portfolio for each team member. Will be useful to keep everything updated as you progress with the project.

https://nus-cs2103-ay1819s1.github.io/cs2103-website/admin/project-deliverables.html#deliverable-project-portfolio-page-ppp

asiddharth replied:
This comment again.

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -833,58 +833,325 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+.  Privacy of user groups should be ensured - developers should not be able to view the contacts or groups or events of the user.
+.  Capacity: Each user should be able to have up to 50 groups that the user can be in at any one point in time.
+.  User activities should be logged in a separate file.
+.  Functions should be testable.
```
What does "testable" mean?

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -833,58 +833,325 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+. User is redirected to app homepage.
++
+
+Use case ends.
```
Good job on use cases and user stories!

asiddharth reviewed (commented):
Good job overall! minor comments included.

asiddharth commented on README.adoc:
```diff
@@ -1,10 +1,9 @@
+https://travis-ci.org/CS2103-AY1819S1-W13-2/main[image:https://travis-ci.org/CS2103-AY1819S1-W13-2/main.svg?branch=master[Build Status]]
+https://ci.appveyor.com/project/kaitingpeck/main[image:https://ci.appveyor.com/api/projects/status/5mecq596nyw31vnu?svg=true[Build status]]
+https://coveralls.io/github/CS2103-AY1819S1-W13-2/main?branch=master[image:https://coveralls.io/repos/github/CS2103-AY1819S1-W13-2/main/badge.svg?branch=master[Coverage Status]]
 https://gitter.im/se-edu/Lobby[image:https://badges.gitter.im/se-edu/Lobby.svg[Gitter chat]]
```
This should be corrected/removed.

asiddharth commented on README.adoc:
```diff
@@ -15,24 +14,36 @@ ifndef::env-github[]
+** Work in multiple groups conveniently and contact multiple persons with one command via a third-party application.
+** Obtain suggested locations and timings for events or meetings based on participants’ locations and timetable.
+** Send automated RSVP requests, for event or meeting participants using the app.
+
```
Good job on : 
1. Adding UI Mockup
2. Modifying home page for the product.
3. Acknowledging AB4
4. Adding correct links for travis.

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -48,33 +48,40 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 ====
 
-=== Viewing help : `help`
+=== Viewing help : `help`, `h`
```
Could indicate that the alias of the command is 'h'.

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -48,33 +48,40 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-=== Viewing help : `help`
+=== Viewing help : `help`, `h`
 
 Format: `help`
```
May add examples for aliases. Same applies for all the following commands.

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -229,11 +236,267 @@ Format: `exit`
 
-_{explain how the user can enable/disable data encryption}_
-// end::dataencryption[]
+* `notification 2 disable`
```
Recommended to add v2.0 features. Other features look good!

asiddharth commented on src/main/java/seedu/address/logic/commands/AddEventCommand.java:
```diff
@@ -0,0 +1,77 @@
+    public CommandResult execute(Model model, CommandHistory commandHistory) throws CommandException {
+        requireNonNull(model);
+
+        /*
```
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -833,60 +833,323 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+* 1a. AddressBook detects a clash with the user's schedule.
++
+** 1a1. AddressBook shows a message informing the user of the clash.
```
1a1 and 1a2 can be combined

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -833,60 +833,323 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+** 1a1. AddressBook shows a message informing the user of the clash.
+** 1a2. AddressBook asks the user if he or she wants to add the event anyway.
++
+If user does not want to add the event, use case ends. Else, use case resumes from step 2.
```
should be step 1a3

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -833,60 +833,323 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+. User requests to add event.
+. AddressBook records and displays event.
+. AddressBook [underline]#notifies user (UC18)# one day before the event.
```
UC 18 is not yet present in the DG. Also, eash step should alternate between the user and EventPlus+ as the actor.

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -833,60 +833,323 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+*MSS*
+
+. User requests to add new group of persons
+. AddressBook creates a new group with that name and displays the newly created group
```
I think this should be EventPlus+

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -833,60 +833,323 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+.  The system should be able to work on 32-bit and 64-bit environments.
+.  The system should respond to the user’s input command within 5 seconds.
+.  The system is not required to handle conflicts between group members’ schedules which are not stored in any online platform (recorded only in a paper diary, etc.)
+.  UI should be aesthetically pleasing i.e. colour, font, layout, pictorial-wise.
```
This NFR needs to be a little bit more specific. This may be done by defining what  aesthetically pleasing is supposed to mean.

asiddharth commented on src/main/java/seedu/address/model/event/Event.java:
```diff
@@ -0,0 +1,91 @@
+    private final String address;
+
+    /**
+     * Every field must be present and not null.
```
Header Comments should specify the functionality of the function.

asiddharth commented on src/test/java/seedu/address/model/event/EventTest.java:
```diff
@@ -0,0 +1,62 @@
+
+        // different address -> returns false
+        editedDoctorAppt = new ScheduledEventBuilder(DOCTORAPPT).withAddress(VALID_ADDRESS_MEETING).build();
+        assertFalse(DOCTORAPPT.equals(editedDoctorAppt));
```
Good job on adding lots of tests and keeping the coverage over 90%

asiddharth reviewed (commented):
Here are some comments : 
1. Change the description to match your product.
![13-2_desc](https://user-images.githubusercontent.com/23400124/46572982-5105b600-c9c1-11e8-9800-87f9f1349a0a.png)
2. May be good to change the name of the jar file.
![13-2_ug1](https://user-images.githubusercontent.com/23400124/46573118-f9684a00-c9c2-11e8-9744-dfaa2f0f0210.png)
3. Product scope should be changed appropriately.
![13-2_dg1](https://user-images.githubusercontent.com/23400124/46573281-a2b03f80-c9c5-11e8-8d8c-5e2224f676d1.png)

asiddharth reviewed (commented):
Hi Team,
Here are some suggestions for this week.
![w13-2](https://user-images.githubusercontent.com/23400124/47269933-e34bb380-d596-11e8-9eef-ca8f04a96e10.png)
(a) Keeping the build passing after each commit helps!
(b) @pratyayj Would be better to use a photo that shows more of your face.

Some comments regarding DG : 
Your DG looks very similar to that of the parent repository. You might want to consider doing some of the following things.
1. Good work adding sequence diagrams! Please also consider adding architectureand class diagrams specific to your application if any

![w13-2_sq1](https://user-images.githubusercontent.com/23400124/47270026-4db12380-d598-11e8-8b49-63a42e27684f.png)
Stray return arrow?

![w13-2_sq2](https://user-images.githubusercontent.com/23400124/47270060-c87a3e80-d598-11e8-83e7-b07fa37451df.png)
Stray return arrows, Missing return arrows.
![w13-2_sq3](https://user-images.githubusercontent.com/23400124/47270081-0c6d4380-d599-11e8-9e0b-e1ee51859408.png)
All return arrows start at teh end of the function call.
Good work on the implementation details!
2. Many references to AB4. Could help to remove references related to AB4 not relevant to your project application.

Progress seems good! Good luck team!

pratyayj committed:
Final edit to this section

pratyayj committed:
Small edit

pratyayj committed:
Another small edit

pratyayj committed:
Another small edit

pratyayj committed:
Add basic, simple UML diagrams to DeveloperGuide

pratyayj committed:
Begin versioning new sequence diagrams

rtanweik committed:
add failing tests for notification/favourit

cjiefeng committed:
Update config.json

Add yuwei alternative authorNames for reposense

rtanweik committed:
Merge branch 'master' into notification

kaitingpeck committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W13-2/main into event-branch

kaitingpeck committed:
add test cases for `DeleteEventCommandParser`

kaitingpeck committed:
add interface methods into stubs

kaitingpeck committed:
Add test cases for `EditEventAddressCommandParser` and update some other test utils and cases

kaitingpeck committed:
add new commad words into auto-complete list

kaitingpeck committed:
make code changes to model for `EditEventAddressCommand`
- changes in `Model`
- test cases covered in `UniqueEventListTest`

kaitingpeck committed:
implement `EditEventAddressCommand`

asiddharth reviewed (commented):
Some comments regarding DG documentation : 
1. The "Design" section in the DG contains a lot of stuff specific to AB4, in text as well as in figures. If you do not use the same or have refactored these to have different names, please make changes here accordingly.
2. It would also help to add a diagram of your own in this section as well.
3. Good job on adding sequence and activity diagrams for some features. Can add some more for features. Meeting Location and notification features could use some diagrams as well.
4. Control does not alternate between the user and the system. Please rectify this.
![13-2](https://user-images.githubusercontent.com/23400124/48145416-c381f280-e2ed-11e8-952d-85f5ad5451e7.png)
5. Would be good to add more test cases. If not remove "{ more test cases …​ }"
![13-2](https://user-images.githubusercontent.com/23400124/48145536-fdeb8f80-e2ed-11e8-9438-ed9b7efd865f.png)

Good luck with the demo!

kaitingpeck committed:
fix checkstyle

kaitingpeck committed:
refactor `arePrefixesPresent` method

pratyayj committed:
Add additional place IDs for meeting locations

kaitingpeck committed:
checkstyle

kaitingpeck committed:
revert change on `AddEventCommandParser`

kaitingpeck committed:
`AddEventCommand` to take `Address` as an optional value (TBD for non-specified values)
- add tests for `EditEventAddressCommand`
- additional updates: update test comments and additional test case in `DeleteEventCommandTest`
- fill in missing values in `AddEventCommandParserTest` test cases and update based on code changes in this commit
- make all addresses distinct in `TypicalEvents` events

kaitingpeck commented:
Thanks for the comments, will update the documentation with reference to these pointers!

kaitingpeck committed:
checkstyle

kaitingpeck committed:
update user guide with optional address field for adding events

kaitingpeck committed:
rename variables

kaitingpeck committed:
rename method

cjiefeng committed:
Fix import contacts test typo, change csv name header and phone number regex

kaitingpeck committed:
update developer guide

kaitingpeck committed:
fix diagram

rtanweik committed:
unfinished ToggleCommand and failing ParserTest for Notification and Favourite

rtanweik committed:
improve abstraction of FavouriteCommand

pratyayj committed:
Add random generator for meeting location

pratyayj committed:
Update status bar message for successful generation

cjiefeng committed:
Update successful import contacts message

cjiefeng committed:
Merge branch 'master' into readcontact-branch

cjiefeng committed:
Update import contacts invalid/empty csv file warning message

cjiefeng committed:
Update import contacts user guide

cjiefeng committed:
Update adding a person user guide

cjiefeng committed:
Add comments for invalid entry in import contacts csv

cjiefeng committed:
Fix checkstyle

cjiefeng committed:
Fix checkstyle

pratyayj committed:
Edit GenerateLocationCommand and its Parser to take in date and time args

pratyayj committed:
Fix Checkstyle errors

pratyayj committed:
Add first update to DeveloperGuide for generateLocation

pratyayj committed:
Complete generateLocation DeveloperGuide

pratyayj committed:
Update UserGuide to with new generateLocation command args

pratyayj committed:
Update AddCommand usage message

rtanweik committed:
improve abstraction of notification/favourite command

pratyayj committed:
Update use cases and appendix for testing

pratyayj committed:
Shift and edit use cases

kaitingpeck commented:
@asiddharth For point 1, as most of the class names used in our code are unchanged from AB4, the information in the Design section is still valid for us. Would it be possible to understand like for example which area should be changed? Thanks!

pratyayj committed:
Add line at EOF

cjiefeng committed:
FileReader.java: update comments

pratyayj committed:
Add new use case for generating meeting location based on people

cjiefeng committed:
Fix file reader to read entries with and without breaklines

cjiefeng committed:
Add test for FilePath.java

rtanweik committed:
resolve test issues

rtanweik committed:
Merge branch 'notification' of https://github.com/ryantanwk/main into notification

# Conflicts:
#	src/main/java/seedu/address/ui/TabPanel.java

cjiefeng committed:
Fix checkstyle

cjiefeng committed:
Merge pull request #172 from pratyayj/further-update-documentation-branch

Update documentation (UserGuide and DeveloperGuide adocs)

cjiefeng committed:
Merge branch 'master' into import-contacts-user-guide

# Conflicts:
#	docs/UserGuide.adoc

cjiefeng committed:
Merge branch 'master' into readcontact-branch

rtanweik committed:
resolve merge conflict

kaitingpeck committed:
add `DeleteEventCommand` command word into autocomplete list

pratyayj committed:
Merge pull request #185 from cjiefeng/import-contacts-user-guide

User guide import contacts

cjiefeng committed:
Merge branch 'master' into readcontact-branch

kaitingpeck committed:
delegate switching of tabs to just `EventPanelDisplayChangedEvent`

kaitingpeck committed:
update developer docs with `findEvent` and `listEvent` implementation

kaitingpeck committed:
fix formatting

kaitingpeck committed:
add edit event address command

kaitingpeck committed:
add formatting

kaitingpeck committed:
fix table width

kaitingpeck committed:
fix table width

kaitingpeck committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W13-2/main into event-branch

# Conflicts:
#	docs/UserGuide.adoc

pratyayj committed:
Change GenerateLocation's commands and modify parser accordingly

cjiefeng committed:
Update test case for import contacts parser

cjiefeng committed:
Fix checkstyle

pratyayj committed:
Update documentation to show that args for generateLocation have changed

pratyayj committed:
Fix Checkstyle

pratyayj committed:
Fix more Checkstyle

cjiefeng committed:
add import contacts alias command

cjiefeng committed:
Update import contacts user guide alias

pratyayj committed:
Update sequence diagram

pratyayj committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W13-2/main

pratyayj committed:
Merge branch 'master' into further-update-documentation-branch

pratyayj committed:
Update doc

pratyayj committed:
Merge pull request #187 from cjiefeng/import-contacts-user-guide

import contacts user guide alias

pratyayj committed:
Merge pull request #183 from cjiefeng/readcontact-branch

import contacts branch

kaitingpeck committed:
remove irrelevant lines

pratyayj committed:
Add first few tests

pratyayj committed:
Add first few tests

pratyayj committed:
Minor revision

pratyayj committed:
Add tests for EmbedGoogleMaps

kaitingpeck committed:
update `Ui` and `Model` class diagrams

kaitingpeck committed:
update file names

kaitingpeck committed:
update extensions

kaitingpeck committed:
Remove SE-EDU items

kaitingpeck committed:
checkstyle

kaitingpeck committed:
update class diagrams for each component

kaitingpeck committed:
update component class diagram images

kaitingpeck committed:
update diagram

kaitingpeck committed:
update diagram

kaitingpeck committed:
add `findEvent` command

kaitingpeck committed:
Update test cases

kaitingpeck committed:
update missing corrupted data files section

rtanweik committed:
Merge pull request #186 from pratyayj/random-generate-location-dev-branch

Random generation of meeting location

kaitingpeck committed:
rearrange test cases

kaitingpeck committed:
add `SeeEventContacts` command
- corresponding CLI command for checking event contacts
- add test cases for new classes
- changes in `Logic` and `Model`
- add command word in auto-complete list

kaitingpeck committed:
rename methods

kaitingpeck committed:
update developer guide with `seeEventContacts` stuff

kaitingpeck committed:
update user guide

kaitingpeck committed:
update `PERSON_DELETED_MESSAGE` in `EventCard`

kaitingpeck committed:
formatting

kaitingpeck committed:
fix image name

kaitingpeck committed:
fix image name

kaitingpeck committed:
update image widths

kaitingpeck committed:
update image widths

kaitingpeck committed:
make image smaller

kaitingpeck committed:
update test cases

kaitingpeck committed:
add command aliases for event-related commands

kaitingpeck committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W13-2/main into event-branch

# Conflicts:
#	src/main/java/seedu/address/logic/parser/GenerateLocationCommandParser.java

kaitingpeck committed:
checkstyle

kaitingpeck committed:
checkstyle

pratyayj committed:
Add newline at EOF

kaitingpeck committed:
update PPP and remove irrelevant sections in Developer Guide

kaitingpeck committed:
update PPP

pratyayj committed:
Fix checkstyle

pratyayj committed:
Fix checkstyle

pratyayj committed:
Fix last checkstyle

pratyayj committed:
Add ShowLocationCommandTest

pratyayj committed:
Fix Checkstyle

kaitingpeck committed:
update PPP

cjiefeng committed:
Update cjiefeng Project Portfolio Page

cjiefeng committed:
Update import contacts sequence diagram and add tags to user guide and developer guide

cjiefeng committed:
Merge branch 'master' into importcontacts-documentation

# Conflicts:
#	docs/UserGuide.adoc

cjiefeng committed:
link user guide and developer guide contributions

cjiefeng committed:
Update import contacts tag to camelCase

cjiefeng committed:
Update PPP

pratyayj committed:
Add more tests

pratyayj committed:
Fix checkstyle

rtanweik committed:
add tests for favourite/notification command and implement toggle command

cjiefeng committed:
Update developer guide and sequence diagram for import contacts

rtanweik committed:
resolve checkstyle errors

cjiefeng committed:
Update import contacts developer guide

rtanweik committed:
resolve checkstyle errors

rtanweik committed:
resolve checkstyle errors

rtanweik committed:
resolve checkstyle error

rtanweik committed:
resolve checkstyle errors

cjiefeng committed:
Fix checkstyle

cjiefeng committed:
Fix checkstyle

rtanweik committed:
comment out failing legacy tests from AB4. Have contacted lecturer on how to proceed

rtanweik committed:
resolve checkstyle errors

rtanweik committed:
Merge pull request #193 from cjiefeng/jack-PPP

Jack ppp

rtanweik committed:
Merge pull request #192 from cjiefeng/importcontacts-documentation

Importcontacts documentation

cjiefeng committed:
Merge pull request #135 from ryantanwk/notification

solve compatibility of notification/favourite with undo/redo

pratyayj committed:
Add test for GenerateLocationCommand and refractor GenerateLocation

pratyayj committed:
Add test for GenerateLocationCommand and refractor GenerateLocation (2)

pratyayj committed:
Fix checkstyle

pratyayj committed:
Fix checkstyle

cjiefeng committed:
Merge pull request #177 from kaitingpeck/event-branch

Add EditEventAddress command and make Address optional in AddEventCommand

pratyayj committed:
Fix checkstyle

pratyayj committed:
Fix Checkstyle

pratyayj committed:
Fix checkstyle

pratyayj committed:
Fix checkstyle

pratyayj committed:
Fix checkstyle last time

pratyayj committed:
Add documentation and comments, and fix readability of tests

pratyayj committed:
Add Javadocs to important methods, logging, tidy code up

pratyayj committed:
Fix checkstyle - 1

pratyayj committed:
Fix checkstyle - 2

pratyayj committed:
Add and change favicon for application

pratyayj committed:
Fix checkstyle - 3

rtanweik committed:
update documentation for toggle command & resolve failing legacy tests

rtanweik committed:
Merge pull request #191 from pratyayj/tests-development-branch

Add tests to cover newly added classes and modified functionality

kaitingpeck committed:
add use case for view event contacts

kaitingpeck committed:
update use cases and user stories

kaitingpeck committed:
update image widths

Commit:
Merge pull request #195 from kaitingpeck/event-branch

User and developer guide changes

rtanweik committed:
add alias in User guide for notification/favourite command

rtanweik committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W13-2/main into notification

kaitingpeck committed:
fix image widths

kaitingpeck committed:
fix formatting

kaitingpeck committed:
update PPP

kaitingpeck committed:
update docs

Commit:
Merge pull request #196 from kaitingpeck/event-branch

fix image widths in DG and UG

kaitingpeck committed:
add test case for command auto-completion

kaitingpeck committed:
make getEvent image smaller

pratyayj committed:
Final updates to UserGuide.adoc

pratyayj committed:
Minor update to UserGuide.adoc

pratyayj committed:
First update to DeveloperGuide.adoc

pratyayj committed:
First update to DeveloperGuide.adoc

pratyayj committed:
Merge branch 'master' into further-update-documentation-branch

rtanweik committed:
Merge pull request #190 from pratyayj/further-update-documentation-branch

Further update documentation branch - User and Dev guides

pratyayj committed:
Update showLocation sequence diagram

pratyayj committed:
Merge pull request #197 from pratyayj/update-diagram-branch

Update showLocation sequence diagram

pratyayj committed:
Minor update to image file extension

pratyayj committed:
Initial update of PPP

kaitingpeck committed:
update PPP

rtanweik committed:
resolve checkstyle

Commit:
Merge pull request #199 from kaitingpeck/event-branch

Update PPP

Commit:
Merge pull request #194 from ryantanwk/notification

update documentation for toggle command & resolve failing legacy tests

kaitingpeck committed:
update PPP

Commit:
Merge pull request #200 from kaitingpeck/event-branch

update PPP

pratyayj committed:
Second update of PPP

pratyayj committed:
Update UserGuide

pratyayj committed:
Update UserGuide

pratyayj committed:
Update UserGuide

pratyayj committed:
Update UserGuide and PPP

pratyayj committed:
Update guides and PPP

kaitingpeck committed:
update ppp

Commit:
Merge pull request #202 from kaitingpeck/ppp-branch

update ppp

pratyayj committed:
Update PPP

kaitingpeck committed:
update devguide sectioning

pratyayj committed:
Update PPP

pratyayj committed:
Update PPP

pratyayj committed:
Update DevGuide

pratyayj committed:
Update DevGuide

pratyayj committed:
Update DevGuide

kaitingpeck committed:
update dev guide links

kaitingpeck committed:
add check for event same event (but not equals) as target and same event as another event in the list

rtanweik committed:
update Ryan's PPP

pratyayj committed:
Update PPP

rtanweik committed:
resolve documentation checkstyle error

kaitingpeck committed:
add tests

pratyayj committed:
Update version number

pratyayj committed:
Update PPP

kaitingpeck committed:
add instructions for user to use `list` to view all persons again

rtanweik committed:
Merge pull request #204 from ryantanwk/notification

update Ryan's PPP

kaitingpeck committed:
fix link

pratyayj committed:
Merge pull request #198 from pratyayj/ppp_update_branch

Add Pratyay PPP

Commit:
Merge pull request #203 from kaitingpeck/ppp-branch

Update Documentation

kaitingpeck committed:
update xml file name

kaitingpeck committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W13-2/main into event-condition-branch

Commit:
Merge pull request #205 from kaitingpeck/event-condition-branch

Add tests and extra condition checking in UniqueEventList

kaitingpeck committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W13-2/main into event-condition-branch

rtanweik committed:
include tags in PR for notification/favourite/toggle

rtanweik committed:
end tag in toggle

Commit:
Merge pull request #206 from ryantanwk/notification

include tags in PR for notification/favourite/toggle

pratyayj committed:
Update Ryan's PPP

pratyayj committed:
Merge pull request #207 from pratyayj/master

Update Ryan's PPP

pratyayj committed:
Update Ryan's PPP

pratyayj committed:
Merge pull request #208 from pratyayj/master

Update Ryan's PPP

cjiefeng committed:
Update Jack PPP

pratyayj committed:
Merge pull request #209 from cjiefeng/Jack-PPP

Update Jack PPP

kaitingpeck committed:
formatting

Commit:
Merge pull request #210 from kaitingpeck/event-condition-branch

Fix formatting

Commit:
Revert "Fix formatting"

Commit:
Merge pull request #211 from CS2103-AY1819S1-W13-2/revert-210-event-condition-branch

Revert "Fix formatting"

Commit:
Merge pull request #212 from kaitingpeck/ppp-branch

Fix formatting

kaitingpeck committed:
remove extra line

kaitingpeck committed:
update author tags

rtanweik committed:
Merge pull request #213 from kaitingpeck/author-tag-branch

update author tags
